garage/tf/q_functions/continuous_mlp_q_function.py

```diff
@@ -23,7 +22,9 @@ def __init__(self,
                  action_merge_layer=-2,
                  output_nonlinearity=None,
                  input_include_goal=False,
-                 bn=False):
+                 bn=False,
+                 reuse=False,
```

Comment:
You need to make this decision for the user. Is the scope reused or not? Your object controls the graph.